src/utils/ICS24Host.sol

```diff
@@ -115,29 +120,40 @@ library ICS24Host {
             revert IICS24HostErrors.IBCMultiPayloadPacketNotSupported();
```

Comment:
Well, we still only support 1 across the application. I added the full correct handling of multiples here since we talked about that many people will look to this implementation. But we do catch the length issue in ICS26Router, so it could be removed here.